roles/install_operator_prereqs/tasks/main.yml

```diff
@@ -34,34 +35,62 @@
 - name: "Download yq 2.2.1"
   shell: "curl -Lo {{ yq_bin_path }} https://github.com/mikefarah/yq/releases/download/2.2.1/yq_linux_amd64 && chmod +x {{ yq_bin_path }}"
 
-- name: "Check if operator-courier is installed"
-  shell: "operator-courier -v"
-  register: operator_courier_version_result
-  ignore_errors: true
+- name: "Install prerequisites specific to legacy operator manifest testing"
+  block:
+  - name: "Check if operator-courier is installed"
+    shell: "operator-courier -v"
+    register: operator_courier_version_result
+    ignore_errors: true
 
-- name: "Install operator-courier if not already installed"
-  shell: "pip3 install operator-courier=={{ operator_courier_version }}"
-  when:
-    - operator_courier_version_result.rc is defined
-    - operator_courier_version_result.rc != 0
+  - name: "Install operator-courier if not already installed"
+    shell: "pip3 install operator-courier=={{ operator_courier_version }}"
+    when:
+      - operator_courier_version_result.rc is defined
+      - operator_courier_version_result.rc != 0
+
+  - name: "Check if offline-cataloger is installed"
+    shell: "offline-cataloger -h"
+    register: offline_cataloger_help_result
+    ignore_errors: true
 
-- name: "Check if offline-cataloger is installed"
-  shell: "offline-cataloger -h"
-  register: offline_cataloger_help_result
-  ignore_errors: true
+  - name: "Git clone offline-cataloger repo with the offline-cataloger command"
+    git:
+      repo: 'https://github.com/kevinrizza/offline-cataloger.git'
+      dest: /tmp/offline-cataloger
+      version: "master"
+      force: true
+    when:
+      - offline_cataloger_help_result.rc is defined
+      - offline_cataloger_help_result.rc != 0
 
-- name: "Git clone offline-cataloger repo with the offline-cataloger command"
-  git:
-    repo: 'https://github.com/kevinrizza/offline-cataloger.git'
-    dest: /tmp/offline-cataloger
-    version: "master"
-    force: true
+  - set_fact:
+      offline_cataloger_bin_path: "{{ go_bin_path }} run -mod=vendor ./cmd/offline-cataloger/main.go"
+    when:
+      - offline_cataloger_help_result.rc is defined
+      - offline_cataloger_help_result.rc != 0
   when:
-    - offline_cataloger_help_result.rc is defined
-    - offline_cataloger_help_result.rc != 0
+    - operator_dir is defined
+    - operator_dir != ""
 
-- set_fact:
-    offline_cataloger_bin_path: "{{ go_bin_path }} run -mod=vendor ./cmd/offline-cataloger/main.go"
+- name: "Install prerequisites specific to operator bundle image testing"
+  block:
+  - name: "Check if skopeo is installed"
+    shell: "skopeo -h"
+    register: skopeo_help_result
+    ignore_errors: true
+
+  - name: "Install skopeo"
+    package:
+      name: skopeo
+      state: present
+    become: true
+    when:
+      - skopeo_help_result.rc is defined
+      - skopeo_help_result.rc != 0
+
+  - name: "Install umoci"
```

Comment:
Could you do the same as with `skopeo` or `operator courier`?
First check whether binary exist and install only in case of failure.